### PR TITLE
SparkMax Device ID -> core

### PR DESCRIPTION
**Pull request by drorta**

### CODE REVIEW

File: src/main/java/frc/utils/devicewrappers/SparkMaxDeviceID.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package frc.utils.devicewrappers;
+
+import com.revrobotics.CANSparkLowLevel;
+
+public record SparkMaxDeviceID(int id, CANSparkLowLevel.MotorType type) {
```
Comment (YHerm) [inline src/main/java/frc/utils/devicewrappers/SparkMaxDeviceID.java:5]:
```suggestion
public record SparkMaxDeviceID(int ID, CANSparkLowLevel.MotorType type) {
```

File: src/main/java/frc/utils/devicewrappers/SparkMaxDeviceID.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+	public SparkMaxDeviceID(int id) {
```
Comment (YHerm) [inline src/main/java/frc/utils/devicewrappers/SparkMaxDeviceID.java:7]:
```suggestion
	public SparkMaxDeviceID(int ID) {
```

File: src/main/java/frc/utils/devicewrappers/SparkMaxDeviceID.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package frc.utils.devicewrappers;
+
+import com.revrobotics.CANSparkLowLevel;
+
+public record SparkMaxDeviceID(int ID, CANSparkLowLevel.MotorType type) {
+
+	public SparkMaxDeviceID(int ID) {
```
Comment (greenblitz4590) [inline src/main/java/frc/utils/devicewrappers/SparkMaxDeviceID.java:7]: how are people supposed to know it is only for brushless

Comment (YHerm) [reply src/main/java/frc/utils/devicewrappers/SparkMaxDeviceID.java:7]: @drorta

**Review by YoniKiriaty — DISMISSED** [review]

**Review by YHerm — APPROVED** [review]

**Review by GilStein1 — APPROVED** [review]
beseder👍

**Review by maya1414 — APPROVED** [review]

**Review by greenblitz4590 — APPROVED** [review]

**Pull request merged**